### PR TITLE
[terra-tabs] Fix for Responsive Tabs tests inconsistent failures

### LINKED ISSUE #868: Responsive Tabs test appears to be inconsistent
# Bug Report

## Description
We're seeing the responsive tabs test fail periodically on travis CI. Need to investigate and see if theres something we can do to make this test more predictable. 

## Steps to Reproduce

1. Run travis CI build on terra-framework
2. Test for responsive tabs may fail with a slight visual mismatch diff

## Expected Behavior
Tests run predictably 

## Possible Solution
My need to see if we need to add a wait for some of the resize observers to complete before we take a screenshot?

## Environment

* Component Name and Version: terra-tabs

### PR DESCRIPTION
**Pull request by avinashg1994**
### Summary

Significantly reduced the inconsistency of the Responsive Tabs tests.

Resolves #868 

### Additional Details

The failures were mainly with 2 tests:

- Default Tabs test (Less frequent)
- Extended Tabs test (More frequent)

I tried a number of fixes for this issue as mentioned here : https://github.com/cerner/terra-framework/issues/868#issuecomment-531771553

By adding a `Browser.refresh()` for the "Extended Tabs" test and a `1000ms wait time` for "Default Tabs", the failures have been significantly reduced.

### CODE REVIEW

**Comment by bjankord** [comment]
Let's try bumping the wait time up some more.
Current wait time of 500ms is still triggering failures at times.
Screen Shot 2019-09-09 at 12 56 46 PM

**Comment by avinashg1994** [comment]
> Let's try bumping the wait time up some more.
> Current wait time of 500ms is still triggering failures at times.
> Screen Shot 2019-09-09 at 12 56 46 PM

Yes, i agree and have increased the wait time to 1000ms. If it still fails sporadically, ill investigate some more and see if there is another alternative.

File: packages/terra-tabs/src/_CollapsibleTabs.jsx [hunk 1]
```diff
@@ -118,9 +118,9 @@ class CollapsibleTabs extends React.Component {
     let isMenuHidden = true;
     for (let i = 0; i < this.props.children.length; i += 1) {
       const tab = this.container.children[i];
-      const tabMarginLeft = parseFloat(window.getComputedStyle(this.menuRef, null).getPropertyValue('margin-left'));
-      const tabMarginRight = parseFloat(window.getComputedStyle(this.menuRef, null).getPropertyValue('margin-right'));
-      const minWidth = parseFloat(window.getComputedStyle(tab, null).getPropertyValue('min-width'));
+      const tabMarginLeft = parseInt(window.getComputedStyle(this.menuRef, null).getPropertyValue('margin-left'), 10);
```
Comment (nramamurth) [inline packages/terra-tabs/src/_CollapsibleTabs.jsx]: @avinashg1994 Just FYI, this was recently updated to use float to address an issue related to overflow - https://github.com/cerner/terra-framework/pull/822

@supreethmr

Comment (avinashg1994) [reply packages/terra-tabs/src/_CollapsibleTabs.jsx]: Yes, I was aware of that. I just reverted those changes in this PR to check if they were the reason behind the inconsistent wdio test failures for Responsive Tabs.

**Comment by avinashg1994** [comment]
> is there a same issue with Notification Dialog..?

Not sure what is the reason behind the Notification Dialog test failure. I was not able to reproduce that error in my local. If i happen to come across that failure i will look into it.

**Pull request merged**